src/routes/reference/pitch.svx

```diff
@@ -8,4 +8,44 @@ flair: reference
 category: Analyse Sound
 ---
 
+<script>
+  import ResourceLink from '$lib/components/ResourceLink.svelte';
+  import Image from '$lib/components/Image.svelte';
+  import Audio from '$lib/components/Audio.svelte';
+</script>
 
+
+FluCoMa's Pitch object returns both pitch and confidence values. When no pitch can be detected, a pitch of 0 Hz is returned (or -999.0 when the unit is in MIDI note mode).
+
+The confidence output is a value between 0 and 1 indicating how confident the algorithm is _that there is_ a pitch. In effect this can be an estimation of how "noisy" (closer to 0) or "harmonic" (closer to 1) the spectrum is.
```

Comment:
The confidence output is a value between 0.0 and 1.0 and indicates how confident the pitch algorithm is in its estimation. It is a useful piece of information that can help us disregard noisy moments in time (closer to 0), where pitch would be hard to discern, as well as giving us a general indication of how harmonic (closer to 1) the spectrum is